common/flights/src/store_client.rs

```diff
@@ -312,4 +316,29 @@ impl StoreClient {
         let vec = serde_json::from_slice(&put_result.app_metadata)?;
         Ok(vec)
     }
+
+    /// Add User
+    pub async fn add_user(
+        &mut self,
+        _username: &str,
+        _password: &str,
+    ) -> anyhow::Result<AddUserActionResult> {
+        todo!()
+    }
+
+    pub async fn drop_user(&mut self, _username: &str) -> anyhow::Result<DropUserActionResult> {
+        todo!()
+    }
+
+    pub async fn update_user(
+        &mut self,
+        _username: &str,
+        _password: &str,
+    ) -> anyhow::Result<UpdateUserActionResult> {
+        todo!()
+    }
+
+    pub async fn get_user_info(&mut self) -> anyhow::Result<GetUsersInfoActionResult> {
```

Comment:
maybe `get_users_info` or `get_all_users_info` is better.

Reply:
let's use `get_all_users_info`